hcs-encrypted-messages/send-messages.js

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env node
+
+import crypto from 'node:crypto';
+import standardEcies from 'standard-ecies';
+
+import {
+    Client,
+    AccountId,
+    PrivateKey,
+    EvmAddress,
+    PublicKey,
+    TopicMessageSubmitTransaction,
+} from '@hashgraph/sdk';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const HCS_TOPIC_ID = process.env.HCS_TOPIC_ID || '0.0.3745107';
+const eciesOptions = {
+    // use all default options
+    curveName: 'secp256k1',
+};
+let messageCount = 0;
+
+// ensure required environment variables are available
+if (!process.env.OPERATOR_ID || !process.env.OPERATOR_KEY) {
+    throw new Error('Must set OPERATOR_ID and OPERATOR_KEY in .env');
+}
+
+// configure client using environment variables
+const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+const operatorKey = PrivateKey.fromStringED25519(process.env.OPERATOR_KEY);
+const client = Client.forTestnet().setOperator(operatorId, operatorKey);
+
+//entrypoint for execution of this example (called at the bottom of the file)
+async function main() {
+    console.log('Operator Account ID:', operatorId.toString());
+
+    const accounts = new Array(5);
+    for (let idx = 0; idx < 5; ++idx) {
+        const id = process.env[`ACCOUNT_${idx}_ID`];
+        const evmAddress = process.env[`ACCOUNT_${idx}_EVMADDRESS`];
```

Comment:
Are these Env vars documented in the readme?

Reply:
good point, will need to update the README!